manifests/admin.pp

```diff
@@ -40,6 +40,20 @@
 # @param default_admin_sudo_cmnds
 #   The set of commands that ``$admin_group`` should be able to run by default
 #
+# @param admin_sudo_options
+#   A hash of sudo options to give to all admin sudo commands
+#
+# @example
+#   simp::admin::admin_sudo_options:
+#     role: 'unconfined_r'
```

Comment:
We should go ahead and drop in a note that this is generally needed for the transition to `root` but other rules should be added for non-root users.

Reply:
Note added as requested.